app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+  before_filter :set_user, only: [:show, :edit, :update]
+
+  def show
+  end
+
+  def edit
+  end
+
+  def update
+    respond_to do |format|
+      if @user.update_attributes(params[:user])
+        format.html { redirect_to user_path }
+        format.json { render :show, status: :ok, location: @user }
+      else
+        format.html { render :edit }
+        format.json { render json: @user.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+private
+
+  def set_user
```

Comment:
@Mywe4ka seems like this name does not represent what actually method is doing. It would be great to rename it. For example ``` set_user -> find_user ```